components/list/list-controls.js

```diff
@@ -18,20 +18,22 @@ export class ListControls extends SelectionControls {
 		return [super.styles, css`
 			:host {
 				--d2l-selection-controls-background-color: var(--d2l-list-controls-background-color);
+				--d2l-selection-controls-offset: -12px;
```

Comment:
This accomplishes the rightward shift of the selection controls when `extend-separators` isn't applied.